### ISSUE TITLE
Graphics of point object is too big.

### ISSUE THREAD

**Issue by harrisyu** [bjorn/tiled#1910]
The new point object is so handy, but the problem is it's graphics is so big and will overlay each other when all in a small place.
It's better to make it customable, or just use a simple cross and don't scale with map.

**Comment by bjorn** [comment]
Yes, the point object should definitely not scale with the map. The fact that it currently does is related to the way object rendering is implemented, which needs to be refactored a bit.

And indeed the current shape is also not ideal. I agree a cross would make the actual position more clear. But I think the scaling is the main issue.

Thanks for opening an issue about this!
